### PR TITLE
#19 - Bug fix for packages with no versions

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #19 
Fixed bug with Registration resource crashing when no versions of package exist

### CODE REVIEW

**Comment by paulodamaso** [comment]
@g4s8 PLease assign me

**Comment by 0crat** [comment]
This pull request [#65](https://github.com/artipie/nuget-adapter/issues/65) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/279ac5dc-a041-4cd4-ac21-2fd24e8fdb85); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artpie/nuget/http/metadata/RegistrationPage.java [hunk 1]
```diff
@@ -80,6 +80,9 @@
      * @throws IOException In case exception occurred on reading data from repository.
      */
     public JsonObject json() throws IOException {
+        if (this.versions.isEmpty()) {
```
Comment (Vatavuk) [inline src/main/java/com/artpie/nuget/http/metadata/RegistrationPage.java:83]: @olegmoz create a test for this scenario

File: src/main/java/com/artpie/nuget/http/metadata/RegistrationPage.java [hunk 2]
```diff
@@ -80,6 +80,9 @@
...
+            throw new IllegalArgumentException("No versions in the page");
```
Comment (Vatavuk) [inline src/main/java/com/artpie/nuget/http/metadata/RegistrationPage.java:84]: @olegmoz Let's be more specific. `Registration page {id} has no versions specified`

**Comment by olegmoz** [comment]
@Vatavuk made changes by your comments, please review

**Comment by 0crat** [comment]
Job was finished in 38 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@Vatavuk please make sure you will at least 3 issues during next CR, thanks

**Comment by Vatavuk** [comment]
@sereshqua ok

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**